extensions/wikia/Recirculation/services/CakeRelatedContentService.php

```diff
@@ -141,13 +146,36 @@ private function formatTitle(Content $content) {
 
 		if ($content->getContentType() == "Discussion Thread") {
 			return $wgContLang->truncate(
-					"[Discussions] {$content->getTitle()}",
+					$content->getTitle(),
 					self::DISCUSSION_THREAD_TITLE_MAX_LENGTH);
 		}
 
 		return $content->getTitle();
 	}
 
+	private function getAuthor($metadata) {
+		if (array_key_exists("authorName", $metadata)) {
+			return $metadata["authorName"];
+		} else {
+			return "";
+		}
+
+	}
+
+	private function getRecirculationContentType($contentType) {
+		switch ($contentType) {
+			case "Discussion Thread":
+				return "discussions";
+			case "Fandom Article":
+				return "fandom";
+			case "Wiki Article":
+				return "wiki";
+			default:
+				return "undefined";
+		}
```

Comment:
It seems like this should be a method on the `$content` objects. Something like `$content->getSource()` might work.